warden/src/persistence.rs

```diff
@@ -65,6 +77,11 @@ static INSERT_INTO_RANGE_LEASE_QUERY: &str = r#"
     IF NOT EXISTS
 "#;
 
+static INSERT_OR_UPDATE_RANGE_ASSIGNMENT_QUERY: &str = r#"
```

Comment:
probably fine for now but does this need to be fenced to prevent older in-flight writes from clobbering the warden state?

Reply:
Good point. I don't see a Cassandra native way to do this. Any ideas? Or are you thinking some kind of lock/master election mechanism?

Reply:
I think we will ultimately want the warden to acquire a leader lease, we can then wrap this write into a [LWT](https://docs.datastax.com/en/drivers/python/3.2/lwt.html) that checks against the lease's sequence number but I'm ok with a TODO for now

Reply:
Done.